src/main/java/uk/org/okapibarcode/output/SvgRenderer.java

```diff
@@ -120,13 +139,38 @@ public void render(Symbol symbol) throws IOException {
                 writer.append("   \"http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd\">\n");
             }
 
+            // render rotation clockwise
+            int rotateHeight = height;
+            int rotateWidth = width;
+            String transform;
+            switch (rotation) {
+                case 90:
+                    rotateHeight = width;
+                    rotateWidth = height;
+                    transform = " transform=\"rotate(-270) translate(0,-" + rotateWidth + ")\"";
```

Comment:
Can this be `rotate(90)`, or does that cause issues?

Reply:
i think when i tried to transform to +90 i could not get things to work right.